lib/membership/member.js

```diff
@@ -178,6 +178,10 @@ Member.prototype._isLocalOverride = function _isLocalOverride(update) {
         return false;
     }
 
+    if (this.incarnationNumber > update.incarnationNumber) {
+        return false;
```

Comment:
Good point, I have added you (and the rest of the world) a test: https://github.com/uber/ringpop-node/pull/284/commits/f93890b4c0e5bb5e60b0778a5dc5ad9ab4c08e49